source/_patterns/01-molecules/15-loan-sections/07-lenders-teams.mustache

```diff
@@ -51,7 +51,9 @@
 							  data-options="disable_for_touch:true; show_on:large"
 							  title="{{teamName}}<br>{{teamType}}<br>{{numLenders}} of {{numTeamMembers}} members"
 							  class="circle-avatar tip-top"
-							  style="background-image:url({{imgUrl}})">
+							  style="background-image:url({{imgUrl}})"
+							  kvtrackevent="Borrower Profile|click-team-image|{{teamName}}"
```

Comment:
This should probably be on the <a> instead of the <span>

Reply:
@emuvente Would you recommend I change it in this ticket, before I merge?

Reply:
Looking at it again I think this will be fine as it is. We can test this out when testing

Reply:
👍